go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	github.com/go-sql-driver/mysql v1.5.0
 	github.com/go-test/deep v1.0.7
 	github.com/gocql/gocql v0.0.0-20210401103645-80ab1e13e309
-	github.com/golang/protobuf v1.4.2
```

Comment:
@briankassouf or @ncabatoff do you see any potential issues with the protobuf bump here and below?